src/sample_package/launch/rostest.launch

```diff
@@ -1,3 +0,0 @@
-<launch>
-    <node
-</launch>
```

Comment:
This was mistakenly committed in, rostests are defined under test/sample_package_test.test